WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormInput.swift

```diff
@@ -66,20 +119,35 @@ private extension ShippingLabelCustomsFormInput {
                                                          comment: "Title for the return policy in Customs screen of Shipping Label flow")
         static let contentTypeTitle = NSLocalizedString("Content Type",
                                                         comment: "Title for the Content Type row in Customs screen of Shipping Label flow")
+        static let contentExplanationTitle = NSLocalizedString("Content Details",
+                                                               comment: "Title for the Content Details row in Customs screen of Shipping Label flow")
+        static let contentExplanationPlaceholder = NSLocalizedString("Type of contents",
+                                                                     comment: "Placeholder for the Content Details row " +
+                                                                        "in Customs screen of Shipping Label flow")
         static let restrictionTypeTitle = NSLocalizedString("Restriction Type",
                                                             comment: "Title for the Restriction Type row in Customs screen of Shipping Label flow")
+        static let restrictionCommentTitle = NSLocalizedString("Restriction Details",
+                                                               comment: "Title for the Restriction Details row in Customs screen of Shipping Label flow")
+        static let restrictionCommentPlaceholder = NSLocalizedString("Type of restriction",
+                                                                     comment: "Placeholder for the Restriction Details row in " +
+                                                                        "Customs screen of Shipping Label flow")
         static let itnTitle = NSLocalizedString("ITN",
                                                 comment: "Title for the ITN row in Customs screen of Shipping Label flow")
         static let itnPlaceholder = NSLocalizedString("Enter ITN (optional)",
                                                       comment: "Placeholder for the ITN row in Customs screen of Shippling Label flow")
+        static let learnMoreITNText = NSLocalizedString("<a href=\"https://pe.usps.com/text/imm/immc5_010.htm\">Learn more</a> about International Tax Number",
```

Comment:
I think this was a mistake in the designs — according to the link, ITN stands for "Internal Transaction Number" so I think we need to update this anywhere we refer to ITN:

```suggestion
        static let learnMoreITNText = NSLocalizedString("<a href=\"https://pe.usps.com/text/imm/immc5_010.htm\">Learn more</a> about Internal Transaction Number",
```

I confirmed we're using "Internal Transaction Number" already in the Android UI.

Reply:
This has been fixed in f61a9ab37e20171c4fb9ebf561f5fb58497725a6.